bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -10,39 +10,46 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.Locale;
 
+import org.eclipse.smarthome.core.thing.type.ChannelGroupType;
+import org.eclipse.smarthome.core.thing.type.ChannelGroupTypeUID;
 import org.eclipse.smarthome.core.thing.type.ChannelType;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeProvider;
 import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
-import org.eclipse.smarthome.core.thing.type.SystemChannelTypeProvider;
 
 /**
  * Implementation providing default system wide channel types
- * 
+ *
  * @author Ivan Iliev - Initial Contribution
  * @author Chris Jackson - Aded battery level
- * 
+ * @author Dennis Nobel - Changed to {@link ChannelTypeProvider}
+ *
  */
-public class DefaultSystemChannelTypeProvider implements SystemChannelTypeProvider {
+public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
```

Comment:
Is there actually any reason left to have this implemented as a Java class instead of simply putting an XML file with the channel types in ESH-INF of this bundle?